src/core/model_repository_manager.cc

```diff
@@ -1221,8 +1221,13 @@ ModelRepositoryManager::Create(
     // model loading / unloading error will be printed but ignored
     RETURN_IF_ERROR(local_manager->PollAndUpdateInternal(&all_models_polled));
   } else {
+    std::unordered_map<std::string, std::vector<const InferenceParameter*>>
+        models;
+    for (const auto& model_name : startup_models) {
+      models[model_name];
```

Comment:
Is this typo?

Reply:
This is to create a default map element, which is model to load without any additional load parameters